jekyll/_api/v1-reference.md

```diff
@@ -288,6 +285,12 @@ You can retry a build with ssh by swapping "retry" with "ssh":
 
 {{ site.data.api.project_branch | api_endpoint }}
 
+<h2 id="new-project-build">Trigger a new Build by Project (preview)</h2>
+
+<span class='label label-info'>Prerequisite:</span> You must go to your Project Settings in the CircleCI app to [Enable Build Processing (preview)]( {{ site.baseurl }}/2.0/build-processing/). This endpoint does **not** yet support the build_parameters or parallel options that the job-triggering endpoint supports.
```

Comment:
Let's remove the reference to "parallel" -- discussion internally is that we are likely to never add that here, as it doesn't really make sense.

Reply:
removed